src/handlers/gestures/GestureDetector.tsx

```diff
@@ -50,7 +50,7 @@ export type GestureConfigReference = {
     HandlerCallbacks<Record<string, unknown>>[] | null
   > | null;
   firstExecution: boolean;
-  useAnimated: boolean;
+  useReanimatedHook: boolean;
```

Comment:
This setting will be `true` for the following example as well:
```ts
const gesture = Gesture.Pan()
  .onBegin(() => {
    'worklet';
    console.log(_WORKLET, 'onBegin');
  });
```
I'd like to suggest renaming it to something more general related to Reanimated worklets, i.e. without "hook".